minos/api_gateway/discovery/views/endpoint.py

```diff
@@ -0,0 +1,31 @@
+from aiohttp import (
+    web,
+)
+
+from ..database import (
+    MinosRedisClient,
+)
+from ..domain import (
+    Microservice,
+)
+from ..exceptions import (
+    NotFoundException,
+)
+from .router import (
+    routes,
+)
+
+
+@routes.view("/microservices/endpoints/{name}")
+class EndpointView(web.View):
+    async def get(self):
+        name = self.request.match_info["name"]
+
+        redis_client = MinosRedisClient(config=self.request.app["config"])
+
+        try:
+            microservice = Microservice.find_by_endpoint(name, redis_client)
+        except NotFoundException:
+            return web.HTTPNoContent()
+
+        return web.json_response(microservice.__dict__)
```

Comment:
I've a controversial question related with this view: Will it be able to obtain a direct matching  on urls with parameters e.g. `/products/6ceb0704-8ea2-4480-accb-29bcd4c44f1f`?

Reply:
You're right it won't.
In this first version I just intended to deal with endpoints as simple strings. Currently it'll be a task of the api gateway to extract the endpoint's name, but definitely we should move to a more schema compliant way of dealing with that.

Reply:
The trouble at this point is that the api gateway will not know anything about what parts of the url are parameters and what are fixed, so I think that a direct matching will not be possible. We may read about how `aiohttp` performs that operation and follow a similar approach.

Reply:
Another alternative would be to dynamically set and unset the `api-gateway` endpoints according to the `discovery` state. 

I mean, instead of asking the `discovery` about what microservice must resolve a request every time, maybe a periodic task in the background could update the exposed endpoints in the `api-gateway` according to the `discovery` which will know all the available endpoints and what microservices could resolve them.

Reply:
As we've said in our daily, I'll code a solution splitting the structure of the endpoint.